apps/backend/core/configuration/validation/validators/realtime_validators.py

```
@@ -0,0 +1,168 @@
+                },
+            )
+
+        except ImportError:
+            return self.create_result(
+                status=ValidationStatus.WARN,
+                message="Pusher SDK not installed",
+                error="Install with: pip install pusher",
+                latency_ms=(time.time() - start_time) * 1000,
+            )
+
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)